src/views/partials/content.ejs

```diff
@@ -1,13 +1,15 @@
 <main class="products-wrapper">
   <%products.forEach( function( prd ){ %>
-  <article class="product">
-    <figure class="figure">
-      <img src=<%= prd.product_image %> alt=<%= prd.product_name %> >
-      <figcaption class="product-content">
-        <p class="product-price">From $<%= prd.product_price%></p>
-        <h4 class="product-title"><%= prd.product_name %></h4>
-      </figcaption>
-    </figure>
-  </article>
+    <article class="product">
+      <figure class="figure">

```

Comment:
is it possible to put the link at these level so it contains all the children items?

Reply:
I could not put it at the level of the figure, since the scss is using pseudo elements and would deconfigure the entire structure, so I put it at the level of the article